plugins.json

```diff
@@ -1,1165 +1,1165 @@
 [
-  {
```

Comment:
can we continue to use 2-space spacing, otherwise its hard to see what changed here :(